ui/app/AppLayouts/Wallet/stores/RootStore.qml

```diff
@@ -488,16 +488,39 @@ QtObject {
         return root.mainModuleInst.addressWasShown(address)
     }
 
-    function getExplorerUrl() {
+    function getExplorerUrl(networkShortName, contractAddress, tokenId) {
```

Comment:
hmm better use the chainID itself instead of the shortname IMO
 
the Collectible entry exposes that property
![image](https://github.com/status-im/status-desktop/assets/11161531/2cca5216-6d15-46fb-8fdf-1bd8d5f38a3f)

And by using that we don't have to go checking if testnet and Goerli are toggled on or off in the settings, it's explicit already

The Collectible  that way we don't have to check if

Reply:
Hey @dlipicar I guess it better to use short name because it would be the same for testnet and mainnet while the chainIds would differ in both cases. This helps us make the code shorter in this case.